ivadomed/utils.py

```diff
@@ -214,7 +219,8 @@ def save_nii(data_lst, z_lst, fname_ref, fname_out, slice_axis, debug=False):
     arr_pred_ref_space = nib.orientations.apply_orientation(arr_ras, trans_orient)
 
     # create nii
-    nib_pred = nib.Nifti1Image(arr_pred_ref_space, nib_ref.affine)
+    nib_pred = nib.Nifti1Image(np.where(arr_pred_ref_space > 0.5, 1.0, 0.0), nib_ref.affine)
```

Comment:
I would recommend to move this thresholding out of this function or at least to add a parameter for this as this is a key post processing step (and to use the function already implemented in utils?).